src/ProjectMWrapper.cpp

```diff
@@ -59,6 +61,42 @@ void ProjectMWrapper::initialize(Poco::Util::Application& app)
 
         _projectM = projectm_create_settings(&settings, PROJECTM_FLAG_NONE);
 
+        // remove any presets in blocklist.txt
+        std::ifstream infile("blocklist.txt");
```

Comment:
i think there is a local config directory we support somewhere where this should probably live
i think a text file is fine

Reply:
Working on this now.

In `ProjectMWrapper::initialize` I see `application.configDir` set to: 

- `/Users/$user/Library/Preferences/projectMSDL/`. 

However in `ProjectMSDLApplication::initialize` I see it trying to load `projectMSDL.properties` file from: 
- `/Users/$user/Library/Preferences/projectM/`.

Which one is preferred? 

My hunch is that the default `projectMSDL` is ok and we should try to load `projectMSDL.properties` from there.

Reply:
sure your call

Reply:
I tried to address by using existing Poco config management. It's my first time with this framework so not sure if I'm doing it right.